uaaextras/templates/email/body-expiring-password.html

```diff
@@ -11,7 +11,8 @@
             will expire in {{password.daysUntilExpiration}} days.
         </p>
         <p>
-            Please <a href="{{password.loginLink}}">log in</a> with your current password</a> and then <a href="{{password.changeLink}}">change your password</a>.
+            First, <a href="{{password.loginLink}}">log in</a> with your current password.
+            After you login, use this <a href="{{password.changeLink}}">link</a> to change your password.
```

Comment:
I think a list of some type (bulleted, numbered, doesn't really matter) would work better to let folks know there's a process to follow, yeah.